doc/help/kdb-find.md

```diff
@@ -55,7 +55,7 @@ kdb find 'fizz'
 #> user/tests/find/tests/fizz/buzz
 #> user/tests/find/tostfizz
 
-sudo kdb umount /tests/find
```

Comment:
We should umount nevertheless. The umount feature of the shell recorder should only be there to catch incorrectly written shell recorder scripts.

Reply:
> The umount feature of the shell recorder should only be there to catch incorrectly written shell recorder scripts.

I agree. I removed the auto unmount feature in the latest updates. This way an incorrectly written test will always fail.

Reply:
Yes, in general tests should fail if they do not properly clean up everything.

I am currently thinking about writing a tutorial of how to setup a computer with Elektra. In this tutorial it would be wanted that everything stays mounted at the end. It is questionable, however, if this tutorial can be checked via the shell recorder because it has many local dependencies, so maybe it won't be run via the shell recorder anyway.